src/sysen2/peek.631

```diff
@@ -3656,16 +3656,17 @@ KAOS00:	ONUM 3,C
 	6TYPE @JNAME
 	ALIGN 1,22.
 	HRRZ A,@CHSSTA
-	CAILE A,10
-	 MOVEI A,10
+	CAILE A,11
+	 MOVEI A,11
 	6TYPE (A)[SIXBIT/CLOSED/
 		  SIXBIT/LISTEN/
 		  SIXBIT/RFCRCV/
 		  SIXBIT/RFCSNT/
 		  SIXBIT/OPEN  /
 		  SIXBIT/LOS   /
 		  SIXBIT/INCXMT/
-		  SIXBIT/LOWLVL/
+		  SIXBIT/FORIGN/ ;Foreign protocol mode
+		  SIXBIT/BRDSNT/ ;Broadcast sent
```

Comment:
You added two more strings, but only increased the number by one.  Is that right?

Reply:
Previously the state %CSFRN (foreign protocol mode) would be presented as "LOWLVL" which seems arachaic (FORIGN is clearer, in my mind), while anything over %CSFRN was presented as "GARBAG". With the new code, %CSBRS (which is %CSFRN+1) is "BRDSNT" and anything higher is "GARBAG".

Reply:
Oh, right!  Never mind then.